### PR TITLE
Feature/2338 PEF & EOI content changes

### PR DESCRIPTION
**Pull request by paulmsmith**
## Proposed changes

[Ticket](https://trello.com/c/HVWu7VnB/2338-add-pef-and-eoi-content-iterations-to-staging-05-day) 

---

Purpose of changes: Iterations to PEF and EOI forms in staging for /funding frontend 

 

Content changes for PEF: 

 

Change: Have you spoken to anyone at the fund about your idea? 

 

To: Have you spoken to anyone at The Fund about your idea? 

 

 

 

Change: How much funding are you planning to apply to us for? 

 

To: How much funding are you planning to apply for from us? 

 

How much funding from us are you planning to apply for? 

 

 

Project Enquiry submitted page 

 

Content missing: 2 tick boxes and content, see below 

 

Change: 

 

 

To: 

Did you know? 

We run local events and workshops 
 

For potential applicants we also send out information about our grant programmes. 

[check box field] Tick this box if you would like to receive invites to events, or information about our grant programmes. 

[Button] Send preferences 

 

We organise user research 

We run qualitative user research to help us to develop our products and services. This could be from a 20 minute survey to a 2 hour interview. 

[check box field] Tick this box if you would like to be involved in our research, or find out more. 

[Button] Send preferences 

 

 

Content changes for EOI: 

 

Change: Have you spoken to anyone at the fund about your idea? 

 

To: Have you spoken to anyone at The Fund about your idea? 

 

 

Change: What programme outcomes do you want to achieve? 

To: What outcomes do you want to achieve? 

 

 

Change: How long do you think your project will take? 

To: How long do you think the project will take? 

 

 

 

Change: How much is your project likely to cost? 

 

To: How much is the project likely to cost? 

 

 

 

Change: How much funding are you planning to apply to us for? 

 

To: How much funding are you planning to apply for from us? 

 

 

Change: When are you likely to submit a development phase application if invited to do so? 

 

To: When are you likely to submit a funding application, if asked to do so? 

 

Expression of Interest submitted page 

 

Content missing: 2 tick boxes and content, see below 

 

Change content in green alert box from: Your expression of interest has been submitted! 

 

To: Your Expression of Interest has been submitted! 

 

 

Change: 

 

 

To: 

Did you know? 

We run local events and workshops 
 

For potential applicants we also send out information about our grant programmes. 

[check box field] Tick this box if you would like to receive invites to events, or information about our grant programmes. 

[Button] Send preferences 

 

We organise user research 

We run qualitative user research to help us to develop our products and services. This could be from a 20 minute survey to a 2 hour interview. 

[check box field] Tick this box if you would like to be involved in our research, or find out more. 

[Button] Send preferences

### CODE REVIEW

File: app/views/pre_application/expression_of_interest/submitted/show.html.erb [hunk 1]
```diff
@@ -75,24 +75,26 @@
...
+          <%=
+            f.check_box :agrees_to_contact,
+                        {
+                            id: :user_agrees_to_contact,
+                            class: "govuk-checkboxes__input"
+                        },
+                        'true', 'false'
+          %>
+          <%=
+            f.label :agrees_to_contact,
```
Comment (stuartmccoll) [inline app/views/pre_application/expression_of_interest/submitted/show.html.erb:89]: Is this tying up correctly where you've removed the `user_` prefix from the label's symbol? The `id` attribute of the check box is still set to `user_agreed_to_contact`, so not sure if they'll match?

File: app/views/pre_application/expression_of_interest/submitted/show.html.erb [hunk 2]
```diff
@@ -102,27 +104,28 @@
...
+          <%=
+            f.check_box :agrees_to_user_research,
+                        {
+                            id: :user_agrees_to_user_research,
+                            class: "govuk-checkboxes__input"
+                        },
+                        'true', 'false'
+          %>
+          <%=
+            f.label :agrees_to_user_research,
```
Comment (stuartmccoll) [inline app/views/pre_application/expression_of_interest/submitted/show.html.erb:118]: Is this tying up correctly where you've removed the `user_` prefix from the label's symbol? The `id` attribute of the check box is still set to `user_agrees_to_user_research`, so not sure if they'll match?

**Review by stuartmccoll — COMMENTED** [review]
I've left a couple of comments - you might know the answer straight away, if not I think it's worth a quick double check 🙂.

File: app/views/pre_application/expression_of_interest/submitted/show.html.erb (hunk 1 above)
Comment (paulmsmith) [reply app/views/pre_application/expression_of_interest/submitted/show.html.erb:89]: They didn't match but this fixes them and makes them match.

File: app/views/pre_application/expression_of_interest/submitted/show.html.erb (hunk 2 above)
Comment (paulmsmith) [reply app/views/pre_application/expression_of_interest/submitted/show.html.erb:118]: They didn't match but this fixes them and makes them match.

**Review by stuartmccoll — APPROVED** [review]

**Pull request merged**